projects/storefrontlib/src/lib/cms-lib/search-box/search-box.component.ts

```diff
@@ -54,11 +54,11 @@ export class SearchBoxComponent implements OnInit, OnDestroy {
   search = (text$: Observable<string>) =>
```

Comment:
Can `void` be added on line 28?

Reply:
ngOnInit? Void is weird there

Reply:
That's probably not what I meant, it seems that a lot has changed after the merge.